src/scss/global.scss

```
@@ -101,4 +100,58 @@ body {
   clip-path: inset(50%);
   border: 0;
   white-space: nowrap;
-}
\ No newline at end of file
+}
+
+//btns
+.btn {
+  width: fit-content;
+  &.btn-primary {
+    background: var(--col-primary);
+    outline: none;
+    border: none;
+    &:hover {
+      background: var(--col-dark);
+    }
+  }
+  &.btn-secondary {
+    background: var(--col-light-blue);
+    color: var(--col-primary);
+    outline: var(--col-primary) !important;
+    border: var(--col-primary) !important;
+    &:hover {
+      background: var(--col-dark);
+    }
+  }
+}
+
+//custom SCSS
+.figcaption {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+.section {
+  &_py {
+    padding: 100px 0;
+    @media (max-width: 769px) {
+      padding: 50px 0;
+    }
+  }
+}
+
+.content_w {
+  max-width: 900px;
```

```suggestion
  max-width: 900px;
```
--content-width: 900px;